docusaurus/docs/dev-docs/api/plugins/admin-panel-api.md

```diff
@@ -479,17 +401,17 @@ export default {
 
 </details>
 
-#### Accessing data with the `useCMEditViewDataManager` React hook
+#### Accessing data in the Content Manger edit view using the `useCMEditViewDataManager` hook
 
-Once an injection zone is defined, the component to be injected in the Content Manager can have access to all the data of the Edit View through the `useCMEditViewDataManager` React hook.
+Once a component is injected in the Content Manager it can have access to all the data of the Edit View through the `useCMEditViewDataManager` React hook.
```

Comment:
```suggestion
Once a component is injected in the Content Manager, it can access all the Edit View data through the `useCMEditViewDataManager` React hook.
```